applications/luci-app-antiblock/htdocs/luci-static/resources/view/antiblock/form.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+'require view';
+'require form';
+'require tools.widgets as widgets';
+
+return view.extend({
+	render: function () {
+		var m, s, o;
+
+		m = new form.Map('antiblock', 'AntiBlock');
```

Comment:
Also, for i18n purposes, use `_('...')` e.g. `_('AntiBlock')` mutatis mutandis throughout for the display strings.